roles/repository/molecule/default/host_vars/repository1.yml

```diff
@@ -0,0 +1,51 @@
+    dest: "{{ content_folder }}/amps_repo/alfresco-googledrive-repo-enterprise.amp"
+  - url: "{{ nexus_repository.releases }}/integrations/alfresco-googledrive-share/{{ amps.googledrive_share.version }}/alfresco-googledrive-share-{{ amps.googledrive_share.version }}.amp"
+    sha1_checksum: "{{ lookup('url', '{{ nexus_repository.releases }}/integrations/alfresco-googledrive-share/{{ amps.googledrive_share.version }}/alfresco-googledrive-share-{{ amps.googledrive_share.version }}.amp.sha1') }}"
+    dest: "{{ content_folder }}/amps_share/alfresco-googledrive-share.amp"
+  - url: "{{ nexus_repository.enterprise_releases }}/services/sync/alfresco-device-sync-repo/{{ amps.device_sync.version }}/alfresco-device-sync-repo-{{ amps.device_sync.version }}.amp"
+    sha1_checksum: "{{ lookup('url', '{{ nexus_repository.enterprise_releases }}/services/sync/alfresco-device-sync-repo/{{ amps.device_sync.version }}/alfresco-device-sync-repo-{{ amps.device_sync.version }}.amp.sha1', username=lookup('env', 'NEXUS_USERNAME'), password=lookup('env', 'NEXUS_PASSWORD')) }}"
+    dest: "{{ content_folder }}/amps_repo/alfresco-device-sync-repo.amp"
+  - url: "{{ nexus_repository.releases }}/aos-module/alfresco-aos-module/{{ amps.aos_module.version }}/alfresco-aos-module-{{ amps.aos_module.version }}.amp"
+    sha1_checksum: "{{ lookup('url', '{{ nexus_repository.releases }}/aos-module/alfresco-aos-module/{{ amps.aos_module.version }}/alfresco-aos-module-{{ amps.aos_module.version }}.amp.sha1') }}"
+    dest: "{{ content_folder }}/amps_repo/alfresco-aos-module.amp"
+  - url: "{{ nexus_repository.enterprise_releases }}/alfresco-governance-services-enterprise-repo/{{ amps.ags_repo.version }}/alfresco-governance-services-enterprise-repo-{{ amps.ags_repo.version }}.amp"
+    sha1_checksum: "{{ lookup('url', '{{ nexus_repository.enterprise_releases }}/alfresco-governance-services-enterprise-repo/{{ amps.ags_repo.version }}/alfresco-governance-services-enterprise-repo-{{ amps.ags_repo.version }}.amp.sha1', username=lookup('env', 'NEXUS_USERNAME'), password=lookup('env', 'NEXUS_PASSWORD')) }}"
+    dest: "{{ content_folder }}/amps_repo/ags-repo.amp"
+  - url: "{{ nexus_repository.enterprise_releases }}/alfresco-governance-services-enterprise-share/{{ amps.ags_share.version }}/alfresco-governance-services-enterprise-share-{{ amps.ags_share.version }}.amp"
+    sha1_checksum: "{{ lookup('url', '{{ nexus_repository.enterprise_releases }}/alfresco-governance-services-enterprise-share/{{ amps.ags_share.version }}/alfresco-governance-services-enterprise-share-{{ amps.ags_share.version }}.amp.sha1', username=lookup('env', 'NEXUS_USERNAME'), password=lookup('env', 'NEXUS_PASSWORD')) }}"
```

Comment:
need to decide if applying here the same changes of https://github.com/Alfresco/alfresco-ansible-deployment/pull/251 or rework that after merging this PR (probably this pr has higher priority?)

Reply:
Ooch completely forgot about it!
Do you mind adding handling that change in your PR (I don't think one has higher prio given we anyway released 1.2.0 but I think it would make more sense...?